File: deploy/android/src/haxor/net/HTTPRequestTask.java
package haxor.net;
import haxe.root.*;

@SuppressWarnings(value={"rawtypes", "unchecked"})
public  class HTTPRequestTask<T, U> extends haxor.thread.Task
{
	public    HTTPRequestTask(haxe.lang.EmptyObject empty)
	{
		super(haxe.lang.EmptyObject.EMPTY);
	}
	
	
	public    HTTPRequestTask(java.lang.String p_url, java.lang.String p_method, boolean p_binary, java.lang.Object p_data)
	{
		super(haxe.lang.EmptyObject.EMPTY);
		haxor.net.HTTPRequestTask.__hx_ctor_haxor_net_HTTPRequestTask(this, p_url, p_method, p_binary, p_data);
	}
	
	
	public static  <T_c, U_c> void __hx_ctor_haxor_net_HTTPRequestTask(haxor.net.HTTPRequestTask<T_c, U_c> __temp_me160694, java.lang.String p_url, java.lang.String p_method, boolean p_binary, java.lang.Object p_data)
	{
		haxor.thread.Task.__hx_ctor_haxor_thread_Task(__temp_me160694, null, null);
		if (( haxe.lang.StringExt.indexOf(p_url, "./", null) >= 0 )) 
		{
			p_url = haxe.root.StringTools.replace(p_url, "./", haxor.net.Web.root);
		}
		
		__temp_me160694.url = p_url;
		__temp_me160694.binary = p_binary;
		__temp_me160694.method = p_method;
		__temp_me160694.data = p_data;
		__temp_me160694.bytesLoaded = 0;
		__temp_me160694.bytesTotal = 0;
		__temp_me160694.error = "";
		haxor.core.Console.Log(( "HTTPRequest> " + __temp_me160694.url ), 4);
	}
	
	
	public static   java.lang.Object __hx_createEmpty()
	{
		return new haxor.net.HTTPRequestTask<java.lang.Object, java.lang.Object>(((haxe.lang.EmptyObject) (haxe.lang.EmptyObject.EMPTY) ));
	}
	
	
	public static   java.lang.Object __hx_create(haxe.root.Array arr)
	{
		return new haxor.net.HTTPRequestTask<java.lang.Object, java.lang.Object>(haxe.lang.Runtime.toString(arr.__get(0)), haxe.lang.Runtime.toString(arr.__get(1)), haxe.lang.Runtime.toBool(arr.__get(2)), ((java.lang.Object) (arr.__get(3)) ));
	}
	
	
	public  java.lang.String url;
	
	public  T request;
	
	public  U response;
	
	public  boolean binary;
	
	public  java.lang.String method;
	
	public  java.lang.Object data;
	
	public  int bytesLoaded;
	
	public  int bytesTotal;
	
	public  java.lang.String error;
	
	@Override public   void OnStart()
	{
		{
		}
		
	}
	
	
	public   void OnError()
	{
		haxor.core.Console.LogError(( ( "HTTPRequest> Error [" + this.error ) + "]" ), null);
	}
	
	
	@Override public   double __hx_setField_f(java.lang.String field, double value, boolean handleProperties)
	{
		{
			boolean __temp_executeDef161598 = true;
			switch (field.hashCode())
			{
				case 1382331801:
				{
					if (field.equals("bytesTotal")) 
					{
						__temp_executeDef161598 = false;
						this.bytesTotal = ((int) (value) );
						return value;
					}
					
					break;
				}
				
				
				case 1095692943:
				{
					if (field.equals("request")) 
					{
						__temp_executeDef161598 = false;
						this.request = ((T) (((java.lang.Object) (value) )) );
						return ((double) (haxe.lang.Runtime.toDouble(((java.lang.Object) (value) ))) );
					}
					
					break;
				}
				
				
				case -326983600:
				{
					if (field.equals("bytesLoaded")) 
					{
						__temp_executeDef161598 = false;
						this.bytesLoaded = ((int) (value) );
						return value;
					}
					
					break;
				}
				
				
				case -340323263:
				{
					if (field.equals("response")) 
					{
						__temp_executeDef161598 = false;
						this.response = ((U) (((java.lang.Object) (value) )) );
						return ((double) (haxe.lang.Runtime.toDouble(((java.lang.Object) (value) ))) );
					}
					
					break;
				}
				
				
				case 3076010:
				{
					if (field.equals("data")) 
					{
						__temp_executeDef161598 = false;
						this.data = ((java.lang.Object) (value) );
						return value;
					}
					
					break;
				}
				
				
			}
			
			if (__temp_executeDef161598) 
			{
				return super.__hx_setField_f(field, value, handleProperties);
			}
			 else 
			{
				throw null;
			}
			
		}
		
	}
	
	
	@Override public   java.lang.Object __hx_setField(java.lang.String field, java.lang.Object value, boolean handleProperties)
	{
		{
			boolean __temp_executeDef161599 = true;
			switch (field.hashCode())
			{
				case 96784904:
				{
					if (field.equals("error")) 
					{
						__temp_executeDef161599 = false;
						this.error = haxe.lang.Runtime.toString(value);
						return value;
					}
					
					break;
				}
				
				
				case 116079:
				{
					if (field.equals("url")) 
					{
						__temp_executeDef161599 = false;
						this.url = haxe.lang.Runtime.toString(value);
						return value;
					}
					
					break;
				}
				
				
				case 1382331801:
				{
					if (field.equals("bytesTotal")) 
					{
						__temp_executeDef161599 = false;
						this.bytesTotal = ((int) (haxe.lang.Runtime.toInt(value)) );
						return value;
					}
					
					break;
				}
				
				
				case 1095692943:
				{
					if (field.equals("request")) 
					{
						__temp_executeDef161599 = false;
						this.request = ((T) (value) );
						return value;
					}
					
					break;
				}
				
				
				case -326983600:
				{
					if (field.equals("bytesLoaded")) 
					{
						__temp_executeDef161599 = false;
						this.bytesLoaded = ((int) (haxe.lang.Runtime.toInt(value)) );
						return value;
					}
					
					break;
				}
				
				
				case -340323263:
				{
					if (field.equals("response")) 
					{
						__temp_executeDef161599 = false;
						this.response = ((U) (value) );
						return value;
					}
					
					break;
				}
				
				
				case 3076010:
				{
					if (field.equals("data")) 
					{
						__temp_executeDef161599 = false;
						this.data = ((java.lang.Object) (value) );
						return value;
					}
					
					break;
				}
				
				
				case -1388966911:
				{
					if (field.equals("binary")) 
					{
						__temp_executeDef161599 = false;
						this.binary = haxe.lang.Runtime.toBool(value);
						return value;
					}
					
					break;
				}
				
				
				case -1077554975:
				{
					if (field.equals("method")) 
					{
						__temp_executeDef161599 = false;
						this.method = haxe.lang.Runtime.toString(value);
						return value;
					}
					
					break;
				}
				
				
			}
			
			if (__temp_executeDef161599) 
			{
				return super.__hx_setField(field, value, handleProperties);
			}
			 else 
			{
				throw null;
			}
			
		}
		
	}
	
	
	@Override public   java.lang.Object __hx_getField(java.lang.String field, boolean throwErrors, boolean isCheck, boolean handleProperties)
	{
		{
			boolean __temp_executeDef161600 = true;
			switch (field.hashCode())
			{
				case 314785609:
				{
					if (field.equals("OnError")) 
					{
						__temp_executeDef161600 = false;
						return ((haxe.lang.Function) (new haxe.lang.Closure(((java.lang.Object) (this) ), haxe.lang.Runtime.toString("OnError"))) );
					}
					
					break;
				}
				
				
				case 116079:
				{
					if (field.equals("url")) 
					{
						__temp_executeDef161600 = false;
						return this.url;
					}
					
					break;
				}
				
				
				case 327758243:
				{
					if (field.equals("OnStart")) 
					{
						__temp_executeDef161600 = false;
						return ((haxe.lang.Function) (new haxe.lang.Closure(((java.lang.Object) (this) ), haxe.lang.Runtime.toString("OnStart"))) );
					}
					
					break;
				}
				
				
				case 1095692943:
				{
					if (field.equals("request")) 
					{
						__temp_executeDef161600 = false;
						return this.request;
					}
					
					break;
				}
				
				
				case 96784904:
				{
					if (field.equals("error")) 
					{
						__temp_executeDef161600 = false;
						return this.error;
					}
					
					break;
				}
				
				
				case -340323263:
				{
					if (field.equals("response")) 
					{
						__temp_executeDef161600 = false;
						return this.response;
					}
					
					break;
				}
				
				
				case 1382331801:
				{
					if (field.equals("bytesTotal")) 
					{
						__temp_executeDef161600 = false;
						return this.bytesTotal;
					}
					
					break;
				}
				
				
				case -1388966911:
				{
					if (field.equals("binary")) 
					{
						__temp_executeDef161600 = false;
						return this.binary;
					}
					
					break;
				}
				
				
				case -326983600:
				{
					if (field.equals("bytesLoaded")) 
					{
						__temp_executeDef161600 = false;
						return this.bytesLoaded;
					}
					
					break;
				}
				
				
				case -1077554975:
				{
					if (field.equals("method")) 
					{
						__temp_executeDef161600 = false;
						return this.method;
					}
					
					break;
				}
				
				
				case 3076010:
				{
					if (field.equals("data")) 
					{
						__temp_executeDef161600 = false;
						return this.data;
					}
					
					break;
				}
				
				
			}
			
			if (__temp_executeDef161600) 
			{
				return super.__hx_getField(field, throwErrors, isCheck, handleProperties);
			}
			 else 
			{
				throw null;
			}
			
		}
		
	}
	
	
	@Override public   double __hx_getField_f(java.lang.String field, boolean throwErrors, boolean handleProperties)
	{
		{
			boolean __temp_executeDef161601 = true;
			switch (field.hashCode())
			{
				case 1382331801:
				{
					if (field.equals("bytesTotal")) 
					{
						__temp_executeDef161601 = false;
						return ((double) (this.bytesTotal) );
					}
					
					break;
				}
				
				
				case 1095692943:
				{
					if (field.equals("request")) 
					{
						__temp_executeDef161601 = false;
						return ((double) (haxe.lang.Runtime.toDouble(((java.lang.Object) (this.request) ))) );
					}
					
					break;
				}
				
				
				case -326983600:
				{
					if (field.equals("bytesLoaded")) 
					{
						__temp_executeDef161601 = false;
						return ((double) (this.bytesLoaded) );
					}
					
					break;
				}
				
				
				case -340323263:
				{
					if (field.equals("response")) 
					{
						__temp_executeDef161601 = false;
						return ((double) (haxe.lang.Runtime.toDouble(((java.lang.Object) (this.response) ))) );
					}
					
					break;
				}
				
				
				case 3076010:
				{
					if (field.equals("data")) 
					{
						__temp_executeDef161601 = false;
						return ((double) (haxe.lang.Runtime.toDouble(this.data)) );
					}
					
					break;
				}
				
				
			}
			
			if (__temp_executeDef161601) 
			{
				return super.__hx_getField_f(field, throwErrors, handleProperties);
			}
			 else 
			{
				throw null;
			}
			
		}
		
	}
	
	
	@Override public   java.lang.Object __hx_invokeField(java.lang.String field, haxe.root.Array dynargs)
	{
		{
			boolean __temp_executeDef161602 = true;
			switch (field.hashCode())
			{
				case 327758243:
				{
					if (field.equals("OnStart")) 
					{
						__temp_executeDef161602 = false;
						return haxe.lang.Runtime.slowCallField(this, field, dynargs);
					}
					
					break;
				}
				
				
				case 314785609:
				{
					if (field.equals("OnError")) 
					{
						__temp_executeDef161602 = false;
						this.OnError();
					}
					
					break;
				}
				
				
			}
			
			if (__temp_executeDef161602) 
			{
				return super.__hx_invokeField(field, dynargs);
			}
			
		}
		
		return null;
	}
	
	
	@Override public   void __hx_getFields(haxe.root.Array<java.lang.String> baseArr)
	{
		baseArr.push("error");
		baseArr.push("bytesTotal");
		baseArr.push("bytesLoaded");
		baseArr.push("data");
		baseArr.push("method");
		baseArr.push("binary");
		baseArr.push("response");
		baseArr.push("request");
		baseArr.push("url");
		{
			super.__hx_getFields(baseArr);
		}
		
	}
	
	
}

File: deploy/windows/src/haxor/net/HTTPRequestTask.cpp
#include <hxcpp.h>

#ifndef INCLUDED_StringTools
#include <StringTools.h>
#endif
#ifndef INCLUDED_haxor_core_Console
#include <haxor/core/Console.h>
#endif
#ifndef INCLUDED_haxor_core_IDisposable
#include <haxor/core/IDisposable.h>
#endif
#ifndef INCLUDED_haxor_core_IRenderable
#include <haxor/core/IRenderable.h>
#endif
#ifndef INCLUDED_haxor_core_IUpdateable
#include <haxor/core/IUpdateable.h>
#endif
#ifndef INCLUDED_haxor_core_Resource
#include <haxor/core/Resource.h>
#endif
#ifndef INCLUDED_haxor_net_HTTPRequestTask
#include <haxor/net/HTTPRequestTask.h>
#endif
#ifndef INCLUDED_haxor_net_Web
#include <haxor/net/Web.h>
#endif
#ifndef INCLUDED_haxor_thread_Activity
#include <haxor/thread/Activity.h>
#endif
#ifndef INCLUDED_haxor_thread_Task
#include <haxor/thread/Task.h>
#endif
namespace haxor{
namespace net{

Void HTTPRequestTask_obj::__construct(::String p_url,::String p_method,bool p_binary,Dynamic p_data)
{
HX_STACK_FRAME("haxor.net.HTTPRequestTask","new",0x48fd59c1,"haxor.net.HTTPRequestTask.new","haxor/net/HTTPRequestTask.hx",62,0x1884b5d1)
HX_STACK_THIS(this)
HX_STACK_ARG(p_url,"p_url")
HX_STACK_ARG(p_method,"p_method")
HX_STACK_ARG(p_binary,"p_binary")
HX_STACK_ARG(p_data,"p_data")
{
	HX_STACK_LINE(63)
	super::__construct(null(),null());
	HX_STACK_LINE(64)
	int _g = p_url.indexOf(HX_CSTRING("./"),null());		HX_STACK_VAR(_g,"_g");
	HX_STACK_LINE(64)
	if (((_g >= (int)0))){
		HX_STACK_LINE(64)
		::String _g1 = ::StringTools_obj::replace(p_url,HX_CSTRING("./"),::haxor::net::Web_obj::root);		HX_STACK_VAR(_g1,"_g1");
		HX_STACK_LINE(64)
		p_url = _g1;
	}
	HX_STACK_LINE(65)
	this->url = p_url;
	HX_STACK_LINE(66)
	this->binary = p_binary;
	HX_STACK_LINE(67)
	this->method = p_method;
	HX_STACK_LINE(68)
	this->data = p_data;
	HX_STACK_LINE(69)
	this->bytesLoaded = (int)0;
	HX_STACK_LINE(70)
	this->bytesTotal = (int)0;
	HX_STACK_LINE(71)
	this->error = HX_CSTRING("");
	HX_STACK_LINE(72)
	::haxor::core::Console_obj::Log((HX_CSTRING("HTTPRequest> ") + this->url),(int)6);
}
;
	return null();
}

//HTTPRequestTask_obj::~HTTPRequestTask_obj() { }

Dynamic HTTPRequestTask_obj::__CreateEmpty() { return  new HTTPRequestTask_obj; }
hx::ObjectPtr< HTTPRequestTask_obj > HTTPRequestTask_obj::__new(::String p_url,::String p_method,bool p_binary,Dynamic p_data)
{  hx::ObjectPtr< HTTPRequestTask_obj > result = new HTTPRequestTask_obj();
	result->__construct(p_url,p_method,p_binary,p_data);
	return result;}

Dynamic HTTPRequestTask_obj::__Create(hx::DynamicArray inArgs)
{  hx::ObjectPtr< HTTPRequestTask_obj > result = new HTTPRequestTask_obj();
	result->__construct(inArgs[0],inArgs[1],inArgs[2],inArgs[3]);
	return result;}

Void HTTPRequestTask_obj::OnStart( ){
{
		HX_STACK_FRAME("haxor.net.HTTPRequestTask","OnStart",0xf81a8884,"haxor.net.HTTPRequestTask.OnStart","haxor/net/HTTPRequestTask.hx",78,0x1884b5d1)
		HX_STACK_THIS(this)
	}
return null();
}


Void HTTPRequestTask_obj::OnError( ){
{
		HX_STACK_FRAME("haxor.net.HTTPRequestTask","OnError",0xe738dfea,"haxor.net.HTTPRequestTask.OnError","haxor/net/HTTPRequestTask.hx",83,0x1884b5d1)
		HX_STACK_THIS(this)
		HX_STACK_LINE(83)
		::haxor::core::Console_obj::LogError(((HX_CSTRING("HTTPRequest> Error [") + this->error) + HX_CSTRING("]")),null());
	}
return null();
}


HX_DEFINE_DYNAMIC_FUNC0(HTTPRequestTask_obj,OnError,(void))


HTTPRequestTask_obj::HTTPRequestTask_obj()
{
}

void HTTPRequestTask_obj::__Mark(HX_MARK_PARAMS)
{
	HX_MARK_BEGIN_CLASS(HTTPRequestTask);
	HX_MARK_MEMBER_NAME(url,"url");
	HX_MARK_MEMBER_NAME(request,"request");
	HX_MARK_MEMBER_NAME(response,"response");
	HX_MARK_MEMBER_NAME(binary,"binary");
	HX_MARK_MEMBER_NAME(method,"method");
	HX_MARK_MEMBER_NAME(data,"data");
	HX_MARK_MEMBER_NAME(bytesLoaded,"bytesLoaded");
	HX_MARK_MEMBER_NAME(bytesTotal,"bytesTotal");
	HX_MARK_MEMBER_NAME(error,"error");
	::haxor::thread::Activity_obj::__Mark(HX_MARK_ARG);
	HX_MARK_END_CLASS();
}

void HTTPRequestTask_obj::__Visit(HX_VISIT_PARAMS)
{
	HX_VISIT_MEMBER_NAME(url,"url");
	HX_VISIT_MEMBER_NAME(request,"request");
	HX_VISIT_MEMBER_NAME(response,"response");
	HX_VISIT_MEMBER_NAME(binary,"binary");
	HX_VISIT_MEMBER_NAME(method,"method");
	HX_VISIT_MEMBER_NAME(data,"data");
	HX_VISIT_MEMBER_NAME(bytesLoaded,"bytesLoaded");
	HX_VISIT_MEMBER_NAME(bytesTotal,"bytesTotal");
	HX_VISIT_MEMBER_NAME(error,"error");
	::haxor::thread::Activity_obj::__Visit(HX_VISIT_ARG);
}

Dynamic HTTPRequestTask_obj::__Field(const ::String &inName,bool inCallProp)
{
	switch(inName.length) {
	case 3:
		if (HX_FIELD_EQ(inName,"url") ) { return url; }
		break;
	case 4:
		if (HX_FIELD_EQ(inName,"data") ) { return data; }
		break;
	case 5:
		if (HX_FIELD_EQ(inName,"error") ) { return error; }
		break;
	case 6:
		if (HX_FIELD_EQ(inName,"binary") ) { return binary; }
		if (HX_FIELD_EQ(inName,"method") ) { return method; }
		break;
	case 7:
		if (HX_FIELD_EQ(inName,"request") ) { return request; }
		if (HX_FIELD_EQ(inName,"OnStart") ) { return OnStart_dyn(); }
		if (HX_FIELD_EQ(inName,"OnError") ) { return OnError_dyn(); }
		break;
	case 8:
		if (HX_FIELD_EQ(inName,"response") ) { return response; }
		break;
	case 10:
		if (HX_FIELD_EQ(inName,"bytesTotal") ) { return bytesTotal; }
		break;
	case 11:
		if (HX_FIELD_EQ(inName,"bytesLoaded") ) { return bytesLoaded; }
	}
	return super::__Field(inName,inCallProp);
}

Dynamic HTTPRequestTask_obj::__SetField(const ::String &inName,const Dynamic &inValue,bool inCallProp)
{
	switch(inName.length) {
	case 3:
		if (HX_FIELD_EQ(inName,"url") ) { url=inValue.Cast< ::String >(); return inValue; }
		break;
	case 4:
		if (HX_FIELD_EQ(inName,"data") ) { data=inValue.Cast< Dynamic >(); return inValue; }
		break;
	case 5:
		if (HX_FIELD_EQ(inName,"error") ) { error=inValue.Cast< ::String >(); return inValue; }
		break;
	case 6:
		if (HX_FIELD_EQ(inName,"binary") ) { binary=inValue.Cast< bool >(); return inValue; }
		if (HX_FIELD_EQ(inName,"method") ) { method=inValue.Cast< ::String >(); return inValue; }
		break;
	case 7:
		if (HX_FIELD_EQ(inName,"request") ) { request=inValue.Cast< Dynamic >(); return inValue; }
		break;
	case 8:
		if (HX_FIELD_EQ(inName,"response") ) { response=inValue.Cast< Dynamic >(); return inValue; }
		break;
	case 10:
		if (HX_FIELD_EQ(inName,"bytesTotal") ) { bytesTotal=inValue.Cast< int >(); return inValue; }
		break;
	case 11:
		if (HX_FIELD_EQ(inName,"bytesLoaded") ) { bytesLoaded=inValue.Cast< int >(); return inValue; }
	}
	return super::__SetField(inName,inValue,inCallProp);
}

void HTTPRequestTask_obj::__GetFields(Array< ::String> &outFields)
{
	outFields->push(HX_CSTRING("url"));
	outFields->push(HX_CSTRING("request"));
	outFields->push(HX_CSTRING("response"));
	outFields->push(HX_CSTRING("binary"));
	outFields->push(HX_CSTRING("method"));
	outFields->push(HX_CSTRING("data"));
	outFields->push(HX_CSTRING("bytesLoaded"));
	outFields->push(HX_CSTRING("bytesTotal"));
	outFields->push(HX_CSTRING("error"));
	super::__GetFields(outFields);
};

static ::String sStaticFields[] = {
	String(null()) };

#if HXCPP_SCRIPTABLE
static hx::StorageInfo sMemberStorageInfo[] = {
	{hx::fsString,(int)offsetof(HTTPRequestTask_obj,url),HX_CSTRING("url")},
	{hx::fsObject /*Dynamic*/ ,(int)offsetof(HTTPRequestTask_obj,request),HX_CSTRING("request")},
	{hx::fsObject /*Dynamic*/ ,(int)offsetof(HTTPRequestTask_obj,response),HX_CSTRING("response")},
	{hx::fsBool,(int)offsetof(HTTPRequestTask_obj,binary),HX_CSTRING("binary")},
	{hx::fsString,(int)offsetof(HTTPRequestTask_obj,method),HX_CSTRING("method")},
	{hx::fsObject /*Dynamic*/ ,(int)offsetof(HTTPRequestTask_obj,data),HX_CSTRING("data")},
	{hx::fsInt,(int)offsetof(HTTPRequestTask_obj,bytesLoaded),HX_CSTRING("bytesLoaded")},
	{hx::fsInt,(int)offsetof(HTTPRequestTask_obj,bytesTotal),HX_CSTRING("bytesTotal")},
	{hx::fsString,(int)offsetof(HTTPRequestTask_obj,error),HX_CSTRING("error")},
	{ hx::fsUnknown, 0, null()}
};
#endif

static ::String sMemberFields[] = {
	HX_CSTRING("url"),
	HX_CSTRING("request"),
	HX_CSTRING("response"),
	HX_CSTRING("binary"),
	HX_CSTRING("method"),
	HX_CSTRING("data"),
	HX_CSTRING("bytesLoaded"),
	HX_CSTRING("bytesTotal"),
	HX_CSTRING("error"),
	HX_CSTRING("OnStart"),
	HX_CSTRING("OnError"),
	String(null()) };

static void sMarkStatics(HX_MARK_PARAMS) {
	HX_MARK_MEMBER_NAME(HTTPRequestTask_obj::__mClass,"__mClass");
};

#ifdef HXCPP_VISIT_ALLOCS
static void sVisitStatics(HX_VISIT_PARAMS) {
	HX_VISIT_MEMBER_NAME(HTTPRequestTask_obj::__mClass,"__mClass");
};

#endif

Class HTTPRequestTask_obj::__mClass;

void HTTPRequestTask_obj::__register()
{
	hx::Static(__mClass) = hx::RegisterClass(HX_CSTRING("haxor.net.HTTPRequestTask"), hx::TCanCast< HTTPRequestTask_obj> ,sStaticFields,sMemberFields,
	&__CreateEmpty, &__Create,
	&super::__SGetClass(), 0, sMarkStatics
#ifdef HXCPP_VISIT_ALLOCS
    , sVisitStatics
#endif
#ifdef HXCPP_SCRIPTABLE
    , sMemberStorageInfo
#endif
);
}

void HTTPRequestTask_obj::__boot()
{
}

} // end namespace haxor
} // end namespace net
